pkg/build/options.go

```diff
@@ -177,3 +177,11 @@ func WithSBOMDir(dir string) Option {
 		return nil
 	}
 }
+
+// WithPOSIXCapabilities is a functional option for overriding the POSIX capabilities encoded in the binary file.
```

Comment:
Because `WithPOSIXCapabilities` overwrites the capabilities slice, I prefer the `[]Cap` argument.
`...Cap` makes me think that the function will append the capabilities to the slice.
Maybe it's just me who thinks that. Please let me know if that is the case & I'll update the code.